source/ed/data.h

```diff
@@ -52,9 +53,19 @@ namespace ed {
 template <typename T>
 void normalize_uri(std::vector<T*>& vec) {
     std::string prefix = navitia::type::static_data::get()->captionByType(T::type);
+
     for (auto* element : vec) {
-        // Suppression des espaces de l'URI
-        boost::algorithm::replace_all(element->uri, " ", "");
+        if (element->uri.find(' ') != std::string::npos) {
+            if (prefix == "stop_point") {
+                LOG4CPLUS_INFO(log4cplus::Logger::getInstance("log"),
+                               "Keeping spaces for stop_point: " << element->uri);
+            } else {
+                // Suppression des espaces de l'URI
+                boost::algorithm::replace_all(element->uri, " ", "");
+                LOG4CPLUS_INFO(log4cplus::Logger::getInstance("log"),
```

Comment:
logs on the INFO level won't appear on aws, only ERROR and WARN levels are logged

Reply:
hum maybe I am wrong, this is true for kraken, but I am not sure for tyr-worker